ext/syncthing/src/github.com/syncthing/syncthing

```diff
@@ -1 +1 @@
-Subproject commit 668eb7c398c2e8d0b9f7beb319b7789681d7d89f
+Subproject commit 6d280e7b6461d407c90f8706e348393c657e995f
```

Comment:
This seems to be because of [this accidental change](https://github.com/syncthing/syncthing-android/commit/6a686cdb9571980d0b72205c37f785aa510d1c08), your rebase didn't take that change for same reason. I reverted that already, so you won't have to do anything (6d280e is correct).

Static fields can be accessed without an object, just with a class. So this isn't a problem. But as I said, that's something for a seperate PR.